packages/feedback/src/modal/integration.tsx

```diff
@@ -5,12 +5,26 @@ import type {
   FeedbackFormData,
   FeedbackModalIntegration,
   IntegrationFn,
+  User,
 } from '@sentry/types';
 import { h, render } from 'preact';
 import { DOCUMENT } from '../constants';
 import { Dialog } from './components/Dialog';
 import { createDialogStyles } from './components/Dialog.css';
 
+function getUser(): User | undefined {
+  const currentUser = getCurrentScope().getUser();
+  const isolationUser = getIsolationScope().getUser();
+  const globalUser = getGlobalScope().getUser();
+  if (currentUser && Object.keys(currentUser).length) {
+    return currentUser;
+  }
+  if (isolationUser && Object.keys(isolationUser).length) {
+    return isolationUser;
+  }
+  return globalUser;
```

Comment:
Can this also return the empty object?

Reply:
Yes it can. We want to check all scopes to see if any have been set, and if not, it'll return an empty object. We were using null coalescing before, which didn't work for empty objects, and would return the empty object without checking the other scopes.